lib/spack/spack/bootstrap/core.py

```diff
@@ -569,6 +573,11 @@ def ensure_winsdk_external_or_raise() -> None:
     spack.detection.update_configuration(externals, buildable=False)
 
 
+def ensure_win_resources() -> None:
+    spack.util.binary_resource.win_ensure_or_acquire_resource("file")
+    spack.util.binary_resource.win_ensure_or_acquire_resource("gpg")
```

Comment:
Is this used at the moment? Or were you planning on keeping it regardless?

(side note: I'm surprised all the imports added here are still needed: I would have assumed the style checker would complain now that most of them are no longer referenced in this diff)

Reply:
It is unused. That said I was planning on keeping it just so we have some reflection of this "bootstrapping" procedure in the bootstrapping module, mostly so future devs looking for this behavior in the bootstrapping module (which, since we named the config `bootstrap_resources`, seems likely) find this. 

I'm not overly tied to having it there, if you want we can remove.

Reply:
> (side note: I'm surprised all the imports added here are still needed: I would have assumed the style checker would complain now that most of them are no longer referenced in this diff)

I thought it would too. Weirdly, despite not considering them unused, none of the style checks griped when I took those imports out, so I'll just do that.

Reply:
> I was planning on keeping it just so we have some reflection of this "bootstrapping" procedure in the bootstrapping module

That works with me!